### ISSUE TITLE
Missing functions in the PSD file

### ISSUE THREAD

**Issue by kdmhorn** [jpsider/ClarityPS#6]
Hi. Like the module. Thanks for making it.

Looks like the PSD is missing at least two functions as I've been looking to use this to design custom reports.

There is an Add-HtmlTitle, but the Close-HtmlTitle function is not listed in the PSD for export.

There is a Close-Header, but the Add-Header function is not listed for export in the PSD file.

In both cases I found adding the missing functions to the FunctionsToExport line in the PSD file resolved the issue.

**Comment by jpsider** [comment]
You are correct! So those are currently in the Private folder and are not exported.
Take a look at the list here, let me know if you think any others should be move to 'Public' so they are properly exported.
https://github.com/jpsider/ClarityPS/tree/master/ClarityPS/private
Thanks!

**Comment by kdmhorn** [comment]
Is there any reason those functions are private? For purposes of making the module more granular would it make sense to make them all public and let the coder decide how or if to use them?

I'd say at the very least anything that is currently published public as an Add, should have its corresponding Close and vice-versa. (examples from a quick look: Add-Help is public, Close-Help is not, Add-Button is public, close-button is not - there are some others)

**Comment by jpsider** [comment]
When I initially released the module it was paired with ReportCardPS, so the default report only called specific functions, and some of those functions roll some of the close commands together So I didnt have to make everything public. But I can see where if you are just using ClarityPS it makes sense to expose them. I can move everything to public. It should not take very long.

**Comment by kdmhorn** [comment]
Cool...thanks!! Yes - originally came across ClarityPS b/c of the ReportCardPS blog, saw where you were putting together sets and was picking those apart to clean up headers and such. Really do like the module - has dressed up one of my script completion reports (once I figured out the order of things and the individual pieces).

**Comment by jpsider** [comment]
Fantastic! I'm glad someone else is using it! feel free to drop issues for any problems or new things you feel are appropriate!

**Comment by jpsider** [comment]
They are all available in 0.4.2! (Update-Module -Name ClarityPS)
Get-Command -Module clarityps | Measure-Object
Count    : 64
